src/logic/schedule-parser/children-info.parser.ts

```diff
@@ -30,20 +30,26 @@ export class ChildrenInfoParser implements ChildrenInfoProvider {
   }
 
   private generateChildren(raw?: string[][]): number[] {
-    if (!raw || raw.length !== 1) {
-      this.logLoadFileError(
-        "Brak informacji o liczbie dzieci. Przyjęto, że w każdym dniu liczba dzieci wynosi " +
-          this.DEFAULT_CHILDREN_NUMBER
-      );
+    let childrenRow;
 
-      const N = this.metaData.dayCount;
-      const children = Array(N);
-      let i = 0;
+    if (!raw || raw.length !== 1) {
+      if (raw && raw.length === 3 && raw[2][0].toLowerCase() === "dzieci") {
```

Comment:
redundant condition
```suggestion
      if (raw.length === 3 && raw[2][0].toLowerCase() === "dzieci") {
```
check if raw[2] exist before getting first element